src/Sulu/Bundle/SecurityBundle/Entity/UserRepository.php

```diff
@@ -287,9 +287,11 @@ public function findUserBySystem($system)
     private function getUserWithPermissionsQuery()
     {
         return $this->createQueryBuilder('user')
+            ->addSelect('contact')
             ->addSelect('userRoles')
             ->addSelect('role')
             ->addSelect('permissions')
+            ->leftJoin('user.contact', 'contact')
```

Comment:
The problem is that this has impact on every single request in the admin section. If you just need this for a single use case it would probably be the better solution to implement it somewhere else.